### ISSUE TITLE
Coach and Learner views: update `<ReportsPracticeQuizLearnerPage>`

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8516]
### User stories for this issue:

1. As a coach, I want to see a summary of a learner across all quiz attempts for this independent practice quiz in the header, so that I can see their best score and best score time.
2. As a coach, I want to see the correct metadata (score, questions correct, time spent, attempted) about the current attempt I select from the dropdown, so that I can get an overview for how the learner did on this particular attempt.
3. As a coach, I want to see detailed information of up to the 5 most recent quiz attempts for this channel quiz by this learner, so that I can see if there is a trend for this learner.
4. As a coach, I want to see how this individual learner did on each question in the "answer log", along with an icon annotation next to questions where they improved, so that I can see if a learner has improved since the last attempt.

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-21 at 12 02 46 PM](https://user-images.githubusercontent.com/13563002/138350864-a065849e-eef1-4e26-843f-3f86a8885650.png)|This is the view for multiple attempts (in coach). Note that this is analogous to what happens with quizzes that are assigned by a teacher in `<ReportsQuizLearnerPage>`|
|![Screen Shot 2021-10-20 at 8 25 03 PM](https://user-images.githubusercontent.com/13563002/138352107-19908466-8a32-4328-b92b-dfa865d02618.png)|This is the view for the first attempt (in learner). Please note the differences between the visibility of the "practice again" button and the lack of a dropdown menu for this first attempt|

### Notes and changes to current designs:
- [Figma for viewing an independent practice quiz learner report](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=999%3A7526)
- Routing and navigation has already been created for this user story
- All strings required should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`
- IMPORTANT: one string that needs to be created is `(XX%) X minutes/days/weeks/months ago`. See the [multiple attempts frame in Figma](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=913%3A6233) for details.
- There is a difference between a report for a learner who has taken this multiple times vs. a learner who has taken it once. It is not in the Figma for this particular user story, but it is in a learner user story. Please see frames above to see the difference
- For third user story
   - Please note that the strings for the dropdown will be in the common strings file.
   - The score of that attempt should be a percentage that is visible in the dropdown menu
   - Needs to populate metadata for each learner
       - Score
       - Questions correct
       - Time spent
       - Attempted

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

_This issue is done when the checkboxes and Gherkin stories below work_

- [ ] coach should not see the "practice again" button, but learner should
- [ ] there should be a UI difference between the first attempt for a learner and subsequent attempts (both from the coach view and from the learner view)
- [ ] selecting an attempt from the dropdown should give me the related metadata for that attempt
- [ ] coach sees summary for all attempts by an individual learner
```
Feature: Learner reports for independent practice quizzes from lessons
	Scenario: coach sees summary for all attempts by an individual learner
		Given that I am in 'Reports'
			And I click on the 'Lessons' subtab
			And I click on a lesson title
			And I click on the title of an independent practice quiz in the 'Reports' subtab
		When I click on a learner's name
		Then I see in the header, the 'Status', the 'Questions correct', the 'Best score', and the 'Best score time' for that learner
```
- [ ] coach sees summary for all attempts by an individual learner with correct icons for the "Answer log"
```
Feature: Learner reports for independent practice quizzes from lessons
Scenario: coach sees summary for all attempts by an individual learner
		Given that I am in 'Reports'
			And I click on the 'Lessons' subtab
			And I click on a lesson title
			And I click on the title of an independent practice quiz in the 'Reports' subtab
		When I click on a learner's name
		Then I see in the header, the 'Status', the 'Questions correct', the 'Best score', and the 'Best score time' for that learner
			And I see correct metadata ('Score', 'Questions correct', 'Time spent', 'Attempted') for that learner for the attempt chosen in the dropdown
			And I see the appropriate icons for correct and incorrect in "Answer log"

```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8516?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
Updates needed for ExamReport utility to provide data for multiple retries.

**Comment by rtibbles** [comment]
Fixed in #8521
